src/etl/usecase/anonymiser.py

```
@@ -0,0 +1,142 @@
+            print(f"Ошибка: Файл {input_file} не найден.")
+            return
+
+        data = self.__zero_step(data)
+        data = self.__anonymise(data, self.__process_text)
+        data = self.__anonymise(data, self.__process_text_again)
+
+        with open(output_file, 'w', encoding='utf-8') as f:
+            json.dump(data, f, ensure_ascii=False, indent=4)
+
+        print(f"Обработано и сохранено в {output_file}")
+        # вывод найденных сущностей с ключами
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF003)